lib/public/views/Runs/Details/index.js

```diff
@@ -86,37 +82,42 @@ const runDetails = (model) => {
         };
 
         return [
-            h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-primary.w-15.mh2#create', {
-                    onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
-                }, 'Add Log Entries'),
-                h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
-            ]),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-success.w-15.mh2', {
-                    onclick: () => model.runs.updateRunTags(),
-                    title: 'This action will overwrite existing tags',
-                    disabled: !model.runs.selectedTagsChanged(),
-                }, 'Update Tags'),
-                h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
-                    'Add one or multiple tags to the run. ',
-                    '(CTRL + click to select multiple tags).',
+            h('.flex-row', [
+                h('.flex.w-100', [
```

Comment:
It looks like you are assigning full width to this div and the run details page is placed at the end.
Could you try to split the space in `w-50` for both elements?

Because of this, we will also need to have a look at the inside elements such ash the `select` tag dropdown which currently is set to `w-30` of its parent.

Reply:
Fixed